lib/services/ngsiService.js

```diff
@@ -129,7 +129,9 @@ function sendRegistrations(unregister, deviceData, callback) {
         } else if (response && body && response.statusCode === 200) {
             var errorField = getErrorField(body);
 
-            if (errorField) {
+			/* fixed an issue with the check of the errorField code. Now the error is not raised if the errorField code is equal to 200 (ReasonPhrase: OK) */
```

Comment:
Please don't leave this comments here. Fix explanations belong to the CHANGES_NEXT_RELEASE file + Github issues.